packages/design-system/src/components/Skeleton/Skeleton.tsx

```diff
@@ -33,6 +35,11 @@ const Skeleton = forwardRef((props: SkeletonProps, ref: Ref<HTMLSpanElement>) =>
 			return <SkeletonParagraph {...rest} ref={ref} />;
 		}
 
+		case 'input': {
+			const { variant, ...rest } = props;
+			return <SkeletonInput {...rest} ref={ref} />;
+		}
+
```

Comment:
Above
```
const { variant, ...rest } = props;
```
Then in all the cases
```suggestion
		case 'input':
			return <SkeletonInput {...rest} ref={ref} />;

```

Reply:
typescript is not as smart as you are